### ISSUE TITLE
[Bug]: newEntity button doesn't update entity list

### ISSUE THREAD

**Issue by DanielBelmes** [EtherealEngine/etherealengine#9176]
### Operating System

Any

### Browser (if applicable)

Any

### What happened?

Was debugging and found when the addEntity button is pressed the entity list doesn't update

### Steps to reproduce

```shell
Open up studio
Open up entity tree
click add Entity button
See list not update

Furthermore if you close list and rexpand. Entity will be there.
```

**Comment by HexaField** [comment]
solved here https://github.com/EtherealEngine/etherealengine/pull/9151

**Comment by DanielBelmes** [comment]
> solved here #9151

Sweet thank you. Will close early so we don't forget
